background/index.ts

```diff
@@ -17,5 +19,11 @@ SimpleAccount.init({
 }).then((account) => {
   provider.connect(account)
 })
-
+// PasskeyAccount.init({
+//   address: process.env.PLASMO_PUBLIC_PASSKEY_ACCOUNT,
+//   owner: new PasskeyOwnerWebAuthn(),
+//   nodeRpcUrl: process.env.PLASMO_PUBLIC_NODE_RPC_URL
+// }).then((account) => {
+//   provider.connect(account)
+// })
```

Comment:
Maybe we could use `if` and `else if` at this moment. Simple account is the first priority, if it is not presented, then try to use passkey account.

What do you think about it?

Reply:
Certainly! we might avoid uncommenting by using .env environment variables. For instance:
```
if (process.env.PLASMO_PUBLIC_ACCOUNT) {
  SimpleAccount.init({
    address: process.env.PLASMO_PUBLIC_ACCOUNT,
    ownerPrivateKey: process.env.PLASMO_PUBLIC_ACCOUNT_OWNER_PRIVATE_KEY,
    nodeRpcUrl: process.env.PLASMO_PUBLIC_NODE_RPC_URL
  }).then((account) => {
    provider.connect(account)
  })
} else if (process.env.PLASMO_PUBLIC_PASSKEY_ACCOUNT) {
  PasskeyAccount.init({
    address: process.env.PLASMO_PUBLIC_PASSKEY_ACCOUNT,
    owner: new PasskeyOwnerWebAuthn(),
    nodeRpcUrl: process.env.PLASMO_PUBLIC_NODE_RPC_URL
  }).then((account) => {
    provider.connect(account)
  })
}
```

Reply:
It is exactly what I think 👍